index.js

```diff
@@ -0,0 +1,66 @@
+document.addEventListener('DOMContentLoaded', () => {
+    getFilm()
+
+    let review = document.getElementById("review");
+    let list = document.getElementById("list");
+    let form = document.querySelector("form");
+
+    form.addEventListener("submit", (event) => {
+        event.preventDefault();
+        let li = document.createElement("li")
+        li.innerText = review.value
+        list.appendChild(li);
+
+    })
+
+})
+
+
+
+
+const getFilm = () => {
+    const select = document.getElementById('select')
```

Comment:
Codes from line 22 to 27 can go inside displayMovieInfo instead to create options for select with all movies' names.